### ISSUE TITLE
<ibm-tooltip-definition> - button does not have aria-label attribute

### ISSUE THREAD

**Issue by ansumiriam** [IBM/carbon-components-angular#2186]
## Title

<ibm-tooltip-definition> - button does not have aria-label attribute

## Detailed description

 ibm-tooltip-definition has a button which does not have aria-label attribute.

<img width="846" alt="image" src="https://user-images.githubusercontent.com/8025573/177971905-c5092b4f-953c-4f65-a3f4-05ab7f096908.png">

Attaching screenshot after inspecting the component on 
https://angular.carbondesignsystem.com/?path=/story/components-tooltip-definition--basic
